UI/NotificationHandler.cs

```diff
@@ -6,25 +6,39 @@
 using Dalamud.Game.Text.SeStringHandling;
 using Dalamud.Game.Text.SeStringHandling.Payloads;
 using Dalamud.Interface;
+using Dalamud.Interface.Windowing;
 using DeathRecap.Events;
 using ImGuiNET;
 
 namespace DeathRecap.UI;
 
-public class NotificationHandler {
+public class NotificationHandler : Window{
     private readonly DalamudLinkPayload chatLinkPayload;
     private readonly DeathRecapPlugin plugin;
     private Death? popupDeath;
     private bool windowWasMoved;
-
-    public NotificationHandler(DeathRecapPlugin plugin) {
+    
+    public NotificationHandler(DeathRecapPlugin plugin) : base("###deathRecapPopup") {
         this.plugin = plugin;
 
+        PositionCondition = ImGuiCond.FirstUseEver;
+        Position = new Vector2(200, 80);
+
+        SizeConstraints = new WindowSizeConstraints
+        {
+            MinimumSize = new Vector2(200, 80),
+            MaximumSize = new Vector2(200, 80)
+        };
+
+        Flags |= ImGuiWindowFlags.NoResize;
+        Flags |= ImGuiWindowFlags.NoCollapse;
+        Flags |= ImGuiWindowFlags.NoFocusOnAppearing;
+
         chatLinkPayload = Service.PluginInterface.AddChatLinkHandler(0, OnChatLinkClick);
     }
 
     private void OnChatLinkClick(uint cmdId, SeString msg) {
-        plugin.Window.ShowDeathRecap = true;
+        plugin.Window.IsOpen = !plugin.Window.IsOpen;
```

Comment:
not sure why we would want to toggle it here, if I click a different Death link I would want it to jump to that death rather than close the window.

Reply:
Ahh, the idea was to be able to close the window that was opened already if you clicked the same link again.
Perhaps some behavior could be added to accommodate this?